Gruntfile.js

```diff
@@ -71,7 +71,7 @@ module.exports = function (grunt) {
             },
             "node-mac": {
                 "dest"      : "<%= downloads %>",
-                "src"       : "http://nodejs.org/dist/v<%= node.version %>/node-v<%= node.version %>-darwin-x86.tar.gz"
+                "src"       : "http://nodejs.org/dist/v<%= node.version %>/node-v<%= node.version %>-darwin-x64.tar.gz"
```

Comment:
Node doesn't have 32-bit builds for Mac anymore:
![image](https://cloud.githubusercontent.com/assets/1067319/18193072/3fc6dd14-711d-11e6-9249-5e9580c779be.png)